regimes/se/org_identities.go

```diff
@@ -0,0 +1,196 @@
+package se
+
+import (
+	"errors"
+	"regexp"
+	"strings"
+
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/internal"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/validation"
+)
+
+// These are placeholder keys and may be changed according to the needs of the project.
+const (
+	// IdentityKeyOrgNr defines the key for the Swedish Organization Number (Organisationsnummer).
+	IdentityKeyOrgNr cbc.Key = "se-orgnr"
+	// IdentityKeyPersonNr defines the key for the Swedish Person Number (Personnummer).
+	IdentityKeyPersonNr cbc.Key = "se-personnr"
+	// IdentityKeyCoordinationNr defines the key for the Swedish Coordination Number (Samordningsnummer).
+	IdentityKeyCoordinationNr cbc.Key = "se-coordnr"
```

Comment:
The pattern we're going for recently is to use codes as opposed to keys. I'd assume there are local codes for each identity like `ON` or `PN` or something that is used frequently.

Reply:
Let me get back to you on that

Reply:
No need to prefix with the country code. Most cases are already context aware. France has a good example of this: https://github.com/invopop/gobl/blob/main/regimes/fr/fr.go#L16-L22